proposal/prefixed-parent-selector.md

```diff
@@ -0,0 +1,58 @@
+# Prefixed Parent Selector: Draft 2
+
+[Issue](https://github.com/sass/sass/issues/1425)
+
+## Background
+
+When transitioning from Less or Stylus, or just learning Sass, nesting is common.
+
+Everything works just as expected until a `&` is in the beginning of a nested selector.
+
+`"&" may only used at the beginning of a compound selector.` 
+
+Considering that there's been quite a few issues regarding this, and that they keep comming up, 
+and that people expect this to work, I think this would be a valuable addition to Sass.
+
+There's a work-around ([with some limitations](https://github.com/sass/sass/issues/1425#issuecomment-404462836)),
+but it's subpar. It might scare of beginners and/or cause unnecessary headache.
+
+
+## Summary
+
+Prefixed parent selectors will work as expected, solving the frustration of current, and new, users of Sass.
```

Comment:
This line doesn't really help to convey any new information to the reader. The summary should be informal and high-level, but it should still be focused on conveying to a user what the feature will do without anything extraneous.

Reply:
Attempted to address this in: https://github.com/sass/sass/pull/2723/commits/031a6d6ca260b20155f028ee2dd8f232f1e976b5